tests/unittest.py

```diff
@@ -110,7 +110,7 @@ def method_name(orig, *args, **kwargs):
         return orig(*args, **kwargs)
     """
 
-    def _around(code: Callable[P, R]) -> None:
+    def _around(code: Callable[Concatenate[TV, P], R]) -> None:
         name = code.__name__
         orig = getattr(target, name)
 
```

Comment:
Unfortunately I don't think it's correct to use `TV` as the type inside `Concatenate`, because `TV` is already in use for the type of `target`.You could introduce another TypeVar I suppose?

Reply:
Bah I was thinking that this is what's passed in, but it isn't -- you're correct.

Reply:
(Why is this function so complicated?)